index.bs

```diff
@@ -1197,7 +1251,7 @@ function onXRSessionStarted(xrSession) {
 
   loadWebGLResources();
 
-  xrSession.baseLayer = new XRWebGLLayer(xrSession, gl);
+  xrSession.updateRenderState({ baseLayer: new XRWebGLLayer(xrSession, gl) });
```

Comment:
Sample code should include a .then handler with at least a comment describing when it would be good to do something on the resolve.

Reply:
@RafaelCintron has a really great point on this.  Whether we are using an event or a promise, if we can't come up with reasonable sample code for how it would be used, then that's a pretty good sign that we shouldn't include it right now.  It's much easier to add things to a standard than to remove them ;)
And TBH, I expect most people will just look at the frame's render state to decide what to do.  So until we hear otherwise, perhaps we should actually remove the event/promise altogether.

Reply:
(sorry this PR has pingponged so much!)

Reply:
![tenor](https://user-images.githubusercontent.com/805273/51703802-2ff6be80-1fcc-11e9-9d9b-613d868f3f25.gif)

Reply:
(Okay. Yes. That a very good point and I'm always happy to reduces API surface... :grimacing:)